app/services/album_service.js

```diff
@@ -0,0 +1,10 @@
+const fetch = require('node-fetch'),
```

Comment:
Any particular reason to use `node-fetch` instead of [request](https://github.com/request/request) ??